src/main/kotlin/model/AbstractPlayer.kt

```diff
@@ -0,0 +1,49 @@
+package model
+
+abstract class AbstractPlayer(open val name: PlayerName, val playerType: PlayerType) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
abstract class AbstractPlayer(val name: PlayerName, val playerType: PlayerType) {
```

Reply:
반영하였습니다.